### PR TITLE
CPU plugin namespaces renaming

### PR DESCRIPTION
**Pull request by vladislav-volkov**
### Details:
 - CPU Plugin refactoring: file names and namespaces

### Tickets:
 - CVS-74811

### CODE REVIEW

File: src/plugins/intel_cpu/src/itt.h [hunk 1]
```diff
@@ -4,18 +4,20 @@
...
 #include <openvino/itt.hpp>
 
-namespace MKLDNNPlugin {
+namespace ov {
+namespace intel_cpu {
 namespace itt {
 namespace domains {
-    OV_ITT_DOMAIN(MKLDNNPlugin);
+    OV_ITT_DOMAIN(intel_cpu);
     OV_ITT_DOMAIN(MKLDNN_LT);
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/src/itt.h:19]: Shouldn't we rename this domain as well?

Comment (vladislav-volkov) [reply src/plugins/intel_cpu/src/itt.h:19]: done

**Comment by vladimir-dudnik** [comment]
@vladislav-volkov @dmitry-gorokhov why this PR changed Open Model Zoo submodule? What content of OMZ will go to wheel packages?

**Comment by vladislav-volkov** [comment]
> why this PR changed Open Model Zoo submodule?
This shouldn't be changed. It seems to have been accidentally touched during a rebase with master. ):

**Pull request merged**